Dockerfile.ppc64le

```diff
@@ -6,82 +10,38 @@
 # docker build -t docker -f Dockerfile.ppc64le .
 #
 
+
+
 FROM ppc64le/gcc:5.2
 
-# Packaged dependencies
-RUN apt-get update && apt-get install -y \
-	apparmor \
-	aufs-tools \
-	btrfs-tools \
-	build-essential \
-	curl \
-	git \
-	iptables \
-	net-tools \
-	libapparmor-dev \
-	libcap-dev \
-	libsqlite3-dev \
-	mercurial \
-	parallel \
-	python-dev \
-	python-mock \
-	python-pip \
-	python-websocket \
-	--no-install-recommends
+RUN apt-get update && apt-get install -y apparmor aufs-tools btrfs-tools build-essential curl ca-certificates git iptables jq net-tools libapparmor-dev libcap-dev libsqlite3-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-RUN rm -rf /usr/local/lvm2
-RUN git clone --no-checkout git://git.fedorahosted.org/git/lvm2.git /usr/local/lvm2 && cd /usr/local/lvm2 && git checkout -q v2_02_103
-RUN curl -o /usr/local/lvm2/autoconf/config.guess 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'
-RUN curl -o /usr/local/lvm2/autoconf/config.sub 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'
+# Get lvm2 source for compiling statically
```

Comment:
In my own multi-arch building, I've had a lot of decent success with
adding --build="$(gcc
-print-multiarch)" to the ./configure flags, but yeah, I suppose updating
those two files should be harmless and probably often helpful (especially
since when they're not and it errors out, they actually print out a
recommendation that they be updated in this very manner with the latest
versions via these exact URLs).